doc/clustering.md

```diff
@@ -36,15 +37,15 @@ available on your network.
 
 You can now join further nodes to the cluster. Note however that these
 nodes should be brand new LXD instances, or alternatively you should
-clear them up before joining, since any existing data on them will be
-lost.
-
-Run `lxd init` and answer `yes` to the question about whether to use
-clustering. Pick a new node name which must be different from the one
-of the bootstrap node or any other nodes you joined so far. Then pick
-an IP or DNS address for the node and answer `yes` to the question
-about whether you're joining an existing cluster. Pick an address of
-an existing node in the cluster and check the fingerprint that gets
+clear their contents them up before joining, since any existing data 
```

Comment:
I think this is not grammatically correct: "clear their contents them up". Should probably be "clear their contents up".

Reply:
Agreed, thanks.